### PR TITLE
Change aggregation script for PNC forms

### PR DESCRIPTION
**Pull request by emord**
@calellowitz @sravfeyn here's an example of changing the aggregation script to use the new form based tables (step 11 in those docs) after https://github.com/dimagi/commcare-hq/pull/19604 & https://github.com/dimagi/commcare-hq/pull/20059

Tests are going to fail and i'll fix tomorrow

### CODE REVIEW

**Comment by emord** [comment]
new buddy @dannyroberts

**Review by dannyroberts — APPROVED** [review]

**Review by calellowitz — APPROVED** [review]
dont have a lot of context on the changes but looks fine

File: custom/icds_reports/tasks.py [hunk 1]
```diff
@@ -184,6 +186,17 @@ def _aggregate_gm_forms(day):
...
 
+@track_time
+def _aggregate_child_health_pnc_forms(day):
+    state_ids = (SQLLocation.objects
+                 .filter(domain=DASHBOARD_DOMAIN, location_type__name='state')
+                 .values_list('location_id', flat=True))
+
+    agg_date = force_to_date(day)
+    for state_id in state_ids:
+        AggregateChildHealthPostnatalCareForms.aggregate(state_id, force_to_date(agg_date))
```
Comment (kaapstorm) [inline custom/icds_reports/tasks.py]: nit: `force_to_date()` was called 2 lines up. Is there a reason it gets called twice?

**Review by kaapstorm — APPROVED** [review]

**Comment by emord** [comment]
haven't forgotten about this PR, but haven't had the chance to do testing I mentioned earlier (somewhat due to server perf issues last couple of days)

**Review by kaapstorm — APPROVED** [review]

**Pull request merged**